mmdet3d/models/voxel_encoders/voxel_encoder.py

```diff
@@ -486,3 +486,130 @@ def fusion_with_mask(self, features, mask, voxel_feats, coors, img_feats,
         out = torch.max(voxel_canvas, dim=1)[0]
 
         return out
+
+
+@MODELS.register_module()
+class CylindricalVFE(nn.Module):
```

Comment:
This module can also be used for other `voxel-based segmentation task` such as PolarNet. We should rename it for more generic.

Reply:
We'll discuss the name later, any good suggestions?

Reply:
May be renamed to `VoxelSegVFE`?

Reply:
Rename to `SegVFE` for now.